src/views/registration-js/content.php

```diff
@@ -115,33 +121,33 @@
 
 			<?php foreach ( $events as $event_id => $tickets ) : ?>
 				<?php
-					$provider_name  = Tribe__Tickets__Tickets::get_event_ticket_provider( $event_id );
-					$provider_obj   = new $provider_name;
-					$provider_class = $provider_class;
-					$providers      = wp_list_pluck( $tickets, 'provider' );
-					$providers_arr  = array_unique( wp_list_pluck( $providers, 'attendee_object' ) );
-
-					if ( empty( $provider_class ) && ! empty( $providers_arr[ $event_id ] ) ) :
-						$provider_class = 'tribe-tickets__item__attendee__fields__form--' . $providers_arr[ $event_id ];
-					endif;
-
-					$has_tpp = Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT === $provider || in_array( Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT, $providers_arr, true );
+				$provider_obj  = Tribe__Tickets__Tickets::get_event_ticket_provider( $event_id );
+				$providers     = wp_list_pluck( $tickets, 'provider' );
+				$providers_arr = array_unique( wp_list_pluck( $providers, 'attendee_object' ) );
+
+				if (
+					empty( $provider_class )
+					&& ! empty( $providers_arr[ $event_id ] )
+				) :
+					$provider_class = 'tribe-tickets__item__attendee__fields__form--' . $providers_arr[ $event_id ];
+				endif;
+
+				$has_tpp = Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT === $provider || in_array( Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT, $providers_arr, true );
 				?>
 				<div
 					class="tribe-tickets__registration__event"
 					data-event-id="<?php echo esc_attr( $event_id ); ?>"
-					data-is-meta-up-to-date="<?php echo absint( $is_meta_up_to_date ); ?>"
+					data-is-meta-up-to-date="<?php echo absint( $this->get( 'is_meta_up_to_date' ) ); ?>"
 				>
-					<?php $this->template( 'registration/summary/content', array( 'event_id' => $event_id, 'tickets' => $tickets ) ); ?>
+					<?php $this->template( 'registration/summary/content', [ 'event_id' => $event_id, 'tickets' => $tickets ] ); ?>
 
 					<div class="tribe-tickets__item__attendee__fields">
 
-						<?php $this->template( 'registration-js/attendees/error', array( 'event_id' => $event_id, 'tickets' => $tickets ) ); ?>
+						<?php $this->template( 'registration-js/attendees/error', [ 'event_id' => $event_id, 'tickets' => $tickets ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: cbeea0a911ce0f9af7553ee01c3eaf1f</sub>